docs/configuration.md

```diff
@@ -238,12 +239,14 @@ Therefore, the following is a valid configuration file:
   "bot": {
     "username": "bot-username",
     "privateKey": {
-      "path": "${user.home}/rsa-private-key.pem"
+      "path": "${HOME}/rsa-private-key.pem"
     }
   }
 }
 ```
 Please mind that if you want to escape the `$` sign, `$${value}` will be replaced by `${value}`.
+And as the matching of environment variables is done after java system properties, if you have a system property with **value** `${value}` and en environment variable with **key** `value`, it will substitute the value of the environment variable
```

Comment:
```suggestion
And as the matching of environment variables is done after Java system properties, if you have a system property with **value** `${value}` and en environment variable with **key** `value`, it will substitute the value of the environment variable
```